### ISSUE TITLE
Error with greek letters in pdf export when using usetex=True and mathptmx

### ISSUE THREAD

**Issue by markushartmann** [matplotlib/matplotlib#16087]
When I export a pdf with savefig I have errors in the greek letters, but only in the pdf the png export is fine. I use the option "usetex=True" and the package "mathptmx". 

```python

import matplotlib.pyplot as plt
from matplotlib import rc
import matplotlib

rc('text', usetex=True)
rc('text.latex',preamble= [
                           r"\usepackage{mathptmx}",
                           ])

x = range(10)

plt.figure()
plt.plot(x, label = r'$\varphi$')
plt.legend()
plt.xlabel('abcdefgh')

name = matplotlib.__version__
plt.savefig('mpl_'+name+'.pdf')
plt.savefig('mpl_'+name+'.png')

plt.show()
```

With the provided code you will get a png and a pdf file. In the pdf file the \varphi symbol in the legend is only a long line. With version 2.2.4 and 1.5.3 this happens. With Version 1.4.3 the pdf and the png is fine. 

**Matplotlib version**

  * Operating system: MacOS 10.12.6
  * Matplotlib version: 2.2.4, 1.5.3 and 1.4.3 tested
  * Matplotlib backend: MacOSX 
  * Python version: 2.7.17

All instalations done via macports. 

Attached the output files:

Version 1.4.3
[mpl_1.4.3.pdf](https://github.com/matplotlib/matplotlib/files/4021409/mpl_1.4.3.pdf)
![mpl_1 4 3](https://user-images.githubusercontent.com/10390818/71758068-a4540600-2e9b-11ea-8cbc-5a10effba88f.png)
Version 1.5.3
[mpl_1.5.3.pdf](https://github.com/matplotlib/matplotlib/files/4021410/mpl_1.5.3.pdf)
![mpl_1 5 3](https://user-images.githubusercontent.com/10390818/71758069-a4540600-2e9b-11ea-99bd-b4b8498e3ee3.png)
Version 2.2.4
[mpl_2.2.4.pdf](https://github.com/matplotlib/matplotlib/files/4021411/mpl_2.2.4.pdf)
![mpl_2 2 4](https://user-images.githubusercontent.com/10390818/71758070-a4ec9c80-2e9b-11ea-8267-07aaa69f9894.png)

Screenshot of error in pdf 

![Bildschirmfoto 2020-01-04 um 02 43 20](https://user-images.githubusercontent.com/10390818/71758110-2f350080-2e9c-11ea-86ab-19844f84e9dc.png)

**Comment by tacaswell** [comment]
Can you try with different pdf viewers?  If I download your pdfs and look at them in chrome or okular (or docview in emacs) the look correct.

If I squint, I could believe that the line is a very distorted version of phi.

I took a quick look at the API and release notes for 1.5.0 and nothing jumped out at me as being related.

**Comment by markushartmann** [comment]
With Adobe reader another error is displayed. The reader message is that the font "StandardSymL_Slant_167" could not be extracted (attached Screenshot).

For me it sounds simmilar to issue #11259

![Bildschirmfoto 2020-01-04 um 16 49 04](https://user-images.githubusercontent.com/10390818/71768018-b329cf80-2f12-11ea-8b58-a8973c17bf29.png)

**Comment by markushartmann** [comment]
Any ideas how to debug this problem a little bit more?

**Comment by anntzer** [comment]
attn @arm2arm from #16894 as well.

If anyone has access to Acrobat Pro, can you try running preflight (https://helpx.adobe.com/acrobat/using/analyzing-documents-preflight-tool-acrobat.html) on one of the "bad" pdf docs and see what comes out?

**Comment by arm2arm** [comment]
pdf and report
[test_arman_report.pdf](https://github.com/matplotlib/matplotlib/files/4382305/test_arman_report.pdf)
[test (1).pdf](https://github.com/matplotlib/matplotlib/files/4382307/test.1.pdf)

**Comment by anntzer** [comment]
So Adobe is just complaining about some bad glyph widths.  This can be fixed manually, as I've done in the attached file here: [test.1.pdf](https://github.com/matplotlib/matplotlib/files/4382341/test.1.pdf)

But locally Acrobat DC still fails to open the new file.  Is that also the case for you?

I'm also confused that apparently preflight does extract the glyphs just fine, but for some reason the pdf viewer refuses to display them.

**Comment by arm2arm** [comment]
So same failure, but the MS Edge  can preview that as before.

![image](https://user-images.githubusercontent.com/2283443/77751788-a3f48280-7026-11ea-836d-97d361f4a319.png)

![image](https://user-images.githubusercontent.com/2283443/77751741-8d4e2b80-7026-11ea-9356-388a6856de81.png)

**Comment by anntzer** [comment]
Well, that's annoying because we can't even know why Acrobat complains about that file (as you mention, edge is happy with it; on linux, poppler and mupdf-based readers are fine too, as is chromium).  Any chance you could contact Adobe to ask them what's wrong with the file, or whether it's something on their side?

**Comment by jkseppan** [comment]
This was a weird one! I compared to the output of pdftex, and the only major differences in the embedded font I saw were:

```
/ItalicAngle -9.48090361795083 def
/FontMatrix [0.001 0.0 0.00016700000000000002 0.001 0.0 0.0] readonly def
```

vs. pdftex's

```
/ItalicAngle -9.4809 def
/FontMatrix [0.001 0 0.000167 0.001 0 0 ] readonly def
```

Because Type 1 is a subset of PostScript and PostScript interpreters typically only support single-precision floating point, I suspect that their parsers return some bogus value for `0.00016700000000000002`. Reducing the precision on these numbers seems to help at least with Preview.app.

I'm preparing a patch, but in the meanwhile people with other pdf readers could see if the attached file works better.

[issue16087.pdf](https://github.com/matplotlib/matplotlib/files/4980361/issue16087.pdf)
